### PR TITLE
Add retry option to deployed services

### PR DESCRIPTION
**Pull request by WangLiNaruto**
Fixed #eclipse-xpanse/xpanse#1694
image
image
image
image

image
image
image

image
image
image
image

image
image
image
image
image

image
image

### CODE REVIEW

File: src/components/content/order/create/OrderSubmit.tsx [hunk 1]
```diff
@@ -29,13 +36,20 @@ function OrderSubmit(state: OrderSubmitProps): React.JSX.Element {
...
-        submitDeploymentRequest.data?.serviceId ?? '',
+    const redeployFailedDeploymentQuery = useRedeployFailedDeploymentQuery();
+    const getSubmitLatestServiceOrderStatusQuery = useLatestServiceOrderStatusQuery(
+        submitDeploymentRequest.data?.orderId ?? '',
         submitDeploymentRequest.isSuccess,
-        state.serviceHostingType,
-        [serviceDeploymentState.DEPLOYMENT_SUCCESSFUL, serviceDeploymentState.DEPLOYMENT_FAILED]
+        [taskStatus.SUCCESSFUL, taskStatus.FAILED]
     );
+    const getReDeployLatestServiceOrderStatusQuery = useLatestServiceOrderStatusQuery(
```
Comment (swaroopar) [inline src/components/content/order/create/OrderSubmit.tsx:47]: would it be possible to have just one query for both cases?
Ideally the old query must be automatically replaced with new one as soon as the ID is replaced with  new order ID.

File: src/components/content/order/orderStatus/OrderSubmitStatusAlert.tsx [hunk 2]
```diff
@@ -3,106 +3,253 @@
...
-    deployedServiceDetails,
+    submitDeploymentRequest,
+    redeployFailedDeploymentQuery,
+    getSubmitLatestServiceOrderStatusQuery,
+    getReDeployLatestServiceOrderStatusQuery,
     serviceProviderContactDetails,
-    isPollingError,
+    retryRequest,
+    onClose,
+    optionType,
```
Comment (swaroopar) [inline src/components/content/order/orderStatus/OrderSubmitStatusAlert.tsx:33]: we do not need this extra field. 
The deployment properties must be taken from the old service. The order status service will only return the status of the order. No information about the service.

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Comment by baixinsui** [comment]
LGTM

**Comment by dengyali0125** [comment]
LGTM

**Comment by dengyali0125** [comment]
LGTM

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**